src/Common.fs

```diff
@@ -46,7 +46,28 @@ module private Util =
 
                     $"""<a href="{ref}" title="{title}">{text}</a>"""
 
-            let mops = !!{| heading = heading; link = link |}
+            let listitem =
+                fun text task check ->
```

Comment:
Use let function bindings instead.
Other functions in Marked are also verbose in their syntax.